cpgconv/src/main/scala/za/ac/sun/plume/CpgDomainObjCreator.scala

```diff
@@ -0,0 +1,77 @@
+package za.ac.sun.plume
+
+import io.shiftleft.codepropertygraph.generated.nodes
+
+object CpgDomainObjCreator {
+
+
+  def arrayInitializer(order : Int): nodes.NewArrayInitializer =
+    nodes.NewArrayInitializer(order = order)
+
+  def binding(name : String, signature : String): nodes.NewBinding =
+    nodes.NewBinding(name = name, signature = signature)
+
+  def block(code : String, columnNumber : Int, lineNumber : Int, order : Int, typeFullName : String, argumentIndex : Int) : nodes.NewBlock =
+    nodes.NewBlock(code = code, columnNumber = Some(columnNumber), lineNumber = Some(lineNumber), order = order, typeFullName = typeFullName, argumentIndex = argumentIndex)
+
+  def call(code : String, name : String, columnNumber : Int, lineNumber : Int, order : Int, methodFullName : String, argumentIndex : Int, signature : String, dispatchType : String,
+           dynamicTypeHintFullName: String, typeFullName : String) : nodes.NewCall =
+    nodes.NewCall(code = code, name = name, columnNumber = Some(columnNumber), lineNumber = Some(lineNumber), order = order,
```

Comment:
I'm not quite sure whether we can use `scalafmt` via gradle. That's worth exploring, because the formatting really doesn't look too good here.

Reply:
`scalafmt` works now and ran it.